Makefile

```diff
@@ -0,0 +1,43 @@
+.SUFFIXES:
+
+CC := hugo --minify --gc
```

Comment:
It would be better to save the cache for the next build.

Reply:
fixed at c8e9655